<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyTicketDialog</class>
 <widget class="QDialog" name="BuyTicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>购买门票</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f5f5f5;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="formContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>出生日期：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="birthDateEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>入园日期：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="visitDateEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>票种类型：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="ticketTypeComboBox">
          <item>
           <property name="text">
            <string>一日票</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>二日票</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>购票数量：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="quantitySpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="disableCheckBox">
          <property name="text">
           <string>持有残疾证明</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="cartButtonLayout">
        <item>
         <widget class="QPushButton" name="calculateButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgba(76, 175, 80, 0.7);
    color: white;
    font-weight: bold;
    padding: 8px 15px;
    border-radius: 5px;
    border: 1px solid rgba(76, 175, 80, 0.9);
}
QPushButton:hover {
    background-color: rgba(76, 175, 80, 0.9);
}
QPushButton:pressed {
    background-color: rgba(56, 155, 60, 0.9);
}</string>
          </property>
          <property name="text">
           <string>添加到购物车</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgba(255, 152, 0, 0.7);
    color: white;
    font-weight: bold;
    padding: 8px 15px;
    border-radius: 5px;
    border: 1px solid rgba(255, 152, 0, 0.9);
}
QPushButton:hover {
    background-color: rgba(255, 152, 0, 0.9);
}
QPushButton:pressed {
    background-color: rgba(235, 132, 0, 0.9);
}</string>
          </property>
          <property name="text">
           <string>移除最后一项</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="line">
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="cartContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="cartLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>购物车</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="cartText">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>购物车为空，请先计算票价并添加到购物车</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="buyButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgba(76, 175, 80, 0.7);
    color: white;
    font-weight: bold;
    padding: 10px 20px;
    border-radius: 5px;
    border: 1px solid rgba(76, 175, 80, 0.9);
}
QPushButton:hover {
    background-color: rgba(76, 175, 80, 0.9);
}
QPushButton:pressed {
    background-color: rgba(56, 155, 60, 0.9);
}</string>
          </property>
          <property name="text">
           <string>确认购买</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgba(244, 67, 54, 0.7);
    color: white;
    font-weight: bold;
    padding: 10px 20px;
    border-radius: 5px;
    border: 1px solid rgba(244, 67, 54, 0.9);
}
QPushButton:hover {
    background-color: rgba(244, 67, 54, 0.9);
}
QPushButton:pressed {
    background-color: rgba(224, 47, 34, 0.9);
}</string>
          </property>
          <property name="text">
           <string>清空购物车</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgba(158, 158, 158, 0.7);
    color: white;
    font-weight: bold;
    padding: 10px 20px;
    border-radius: 5px;
    border: 1px solid rgba(158, 158, 158, 0.9);
}
QPushButton:hover {
    background-color: rgba(158, 158, 158, 0.9);
}
QPushButton:pressed {
    background-color: rgba(138, 138, 138, 0.9);
}</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
